diff --git a/principalmapper/util/case_insensitive_dict.py b/principalmapper/util/case_insensitive_dict.py
index ff5ee90..0fd4a99 100644
--- a/principalmapper/util/case_insensitive_dict.py
+++ b/principalmapper/util/case_insensitive_dict.py
@@ -31,7 +31,8 @@
 #      See the License for the specific language governing permissions and
 #      limitations under the License.

-from collections import Mapping, MutableMapping, OrderedDict
+from collections import OrderedDict
+from collections.abc import Mapping, MutableMapping


 class CaseInsensitiveDict(MutableMapping):

